debian/postinst

```diff
@@ -5,4 +5,4 @@
 
 set -e
 
-/opt/one_compiler/bin/one-prepare-venv
+/usr/share/one/bin/one-prepare-venv
```

Comment:
I think you can invoke the `one-prepare-venv` file directly because you already made a symlink in `/usr/bin` directory.